src/index.ts

```diff
@@ -65,4 +77,39 @@ export class MhDynamoDbClient {
     const response = await this.client.send(new BatchGetCommand(args));
     return response;
   }
+
+  getOptions(): Options | undefined {
+    return this.options;
+  }
+
+  private mergeOptions(opt1, opt2): Options | undefined {
```

Comment:
struggling with the naming here

opt1 = `defaultOptions`
opt2 = `options`

I guess: 
`defaultOptions`:  Options
`options`:  Options | undefined

not sure about the first lines that return, is there a case where you call mergeOptions without defaultOptions?